nettoyage.py

```diff
@@ -341,6 +344,15 @@ def correct_date(df):
 
     return df
 
+
+##### Fonction pour mettre en qualité le champ objetMarche
+def replace_char(df):
+
+    # Remplacement brutal du caractère (?) par un espace
+    df['objetMarche'] = df['objetMarche'].str.replace ('�', ' ') 
```

Comment:
Suggestion :
1. donner la representation en byte 
2. la remplacer par "e", le cas plus probable
3. appliquer l'operation pour les colonnes a risques

cols =["objetMarche", "acheteurNom", "denominationSocialeEtablissement"]
for col in cols:
  df[col] = df[col].str.replace ('\00', 'e')

Reply:
oui bonnes idées.
tu penses que \00 correspond bien au caractère � ?

Reply:
Je pense que c'est le caractere nul oui :
https://www.fileformat.info/info/charset/UTF-8/list.htm

Reply:
quand je fais ça sur mon Mac 
```
import pandas as pd
import csv
df = pd.read_csv("decp_augmente.csv")
df2 = df['objetMarche'].str.replace ('�','XX')
df2.to_csv("toto.csv", quoting=csv.QUOTE_NONNUMERIC)
```


ca fonctionne... trop bizarre